api/generic-cti/answer-event.rst

```diff
@@ -0,0 +1,206 @@
+Call answered
+=============
+
+.. include:: /api/generic-cti/cti-endpoint-limitations.include.rst
+
+.. list-table:: Available attributes and sample data for answered events
+   :widths: 20, 40, 40
+   :header-rows: 1
+
+   * - Attribute
+     - Possible value
+     - Description
+   * - ``event``
+     - ``answer``
+     - Tell Zammad there's a new call
+   * - ``from``
+     - e.g. ``493055571600``, ``02214710334``, ``anonymous``
+     - Number that initiated the call
+   * - ``to``
+     - e.g. ``49221470334``, ``03023125771``
+     - Number that is being called
+   * - ``direction``
+     - ``in`` or ``out``
+     - The call direction - if your agent initiates a call this will be ``out``
+   * - ``callId``
+     - e.g. ``53ba82e2bd6d12d9fb2d3838f0cfb070``, ``5fb9532f40da834a``,
+       ``123456789``
+     - This is the CallID from the ``newCall`` event.
+   * - ``answeringNumber``
+     - e.g. ``42``, ``jdoe``, ``jdoe@example.com``, ``3``
```

Comment:
It may be also the complete phone number, depending in the BPX.

``493055571600``, ``02214710334``

Reply:
I'll add 2 full caller IDs, but will use german drama numbers.